types/pino/index.d.ts

```diff
@@ -461,6 +462,10 @@ declare namespace P {
          * The key in the JSON object to use for timestamp display. Default: "time".
          */
         timestampKey?: string;
+        /**
+         * Format output of message, e.g. {level} - {pid} will output message: INFO - 1123 Default: `false`.
+         */
+        messageFormat?: boolean | string;
```

Comment:
Based on [the code](https://github.com/pinojs/pino-pretty/blob/c0c6b52ecd58a3844d276ad1a5d8caa263ce0081/lib/utils.js#L200), it looks like `true` would never be a valid value for this property, so we can narrow the type accordingly:
```suggestion
        messageFormat?: false | string;
```